docs/samples/digital_twin_graph/diagrams/vehicle_graph.puml

```diff
@@ -0,0 +1,78 @@
+@startuml
+
+object Vehicle {
+  model_id: "dtmi:sdv:vehicle;1"
+  instance_id: "550e8400-e29b-41d4-a716-446655440000"
+  vehicle_identification.vin: 1M8GDM9AXKP042788
```

Comment:
out of curiosity, how was this generated? A VIN can be considered PII, so even if this is a mock value it might be better to have some kind of placeholder like all 0's

Reply:
Replaced with all 0's.